client.lua

```diff
@@ -1,25 +1,39 @@
 CreateThread(function()
+	local playerPed = PlayerPedId()

```

Comment:
single non-updating player ped variable -> you will lose it the moment the model changes or player dies (which is inevitable)

Reply:
thats why i updated that anyway, moved to loop